industry_restaurant/data/product_pricelist.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="product_pricelist_1" model="product.pricelist">
+    <field name="name">Default</field>
+  </record>
```

Comment:
Default pricelist is defined twice... I think you need to define it before activating the config setting to avoid the other default one to be created